saleor/product/migrations/0187_auto_20230911_0704.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.20 on 2023-09-11 07:04
+
+import django.contrib.postgres.indexes
+from django.db import migrations, models
+from django.contrib.postgres.operations import AddIndexConcurrently
+
+
+class Migration(migrations.Migration):
+    atomic = False
```

Comment:
Because that migration is not atomic, please split changes into two migrations:
- adding field, and default (which will be atomic)
- adding index concurrenlty in nonatomic migration

Reply:
@kadewu Hi, thank you for you review. The migrations has been updated according to your remarks.